core/commands/log.go

```diff
@@ -146,3 +155,82 @@ This will only return 'info' logs from bitswap and skip 'debug'.
 		return res.Emit(pipeReader)
 	},
 }
+
+var logGetLevelCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Get the logging level.",
+		ShortDescription: `
+'ipfs log get-level' is a utility command used to get the logging level for
+a specific subsystem, the global default, or all subsystems.
+
+This complements 'ipfs log level' and provides runtime visibility into the
+current logging configuration (whether set by GOLOG_LOG_LEVEL environment
+variable or changed dynamically).
+
+Examples:
+  ipfs log get-level       # Show levels for all subsystems
+  ipfs log get-level all   # Show the global default level
+  ipfs log get-level core  # Show the core subsystem level
+`,
```

Comment:
I believe we need to move this over to the existing `level` command since we decided we didn't want a new `get-level` subcommand.

Reply:
Fixed: See updated PR description